src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -171,6 +171,8 @@ public String deploy(AppDeploymentRequest request) {
 				.then(() -> operations.applications()
 					.start(StartApplicationRequest.builder()
 						.name(name)
+						.stagingTimeout(deploymentProperties.getStagingTimeout())
```

Comment:
Not sure I understand. Can't we have a test that asserts stagingTimeout in `StartApplicationRequest` with the `stagingTimeout` being passed as deployment property?